media/js/mozorg/manifesto.js

```diff
@@ -6,11 +6,9 @@ $(function() {
     'use strict';
 
     // Open Twitter in a sub window
-    var openTwitterSubwin = function (section, url) {
-
+    var openTwitterSubwin = function (url) {
         window.dataLayer.push({
```

Comment:
Nit: not introduced in this change, but would be good to fix here. I usually try to put calls to `window.dataLayer.push()` last in functions where it's needed. It's probably the lowest priority piece of code here, and should `window.dataLayer` ever become `undefined` (e.g. accidentally in the future), then the functionality won't break if it comes after the share code.